### PR TITLE
added some null checks

### PR DESCRIPTION
**Pull request by KevinSchildhorn**
Adding some null checks to fix crashes

### CODE REVIEW

**Comment by KevinSchildhorn** [comment]
Fatal Exception: kotlin.UninitializedPropertyAccessException
lateinit property recycler has not been initialized
co.touchlab.sessionize.settings.SettingsFragment.updateContent + 106 (SettingsFragment.kt:106)


Caused by java.lang.NullPointerException
Attempt to read from field 'android.os.Bundle android.app.Notification.extras' on a null object reference
androidx.core.app.NotificationCompat.getExtras + 5042 (NotificationCompat.java:5042)
androidx.core.app.NotificationManagerCompat.notify + 196 (NotificationManagerCompat.java:196)
co.touchlab.sessionize.NotificationPublisher.onReceive + 28 (NotificationPublisher.kt:28)

File: sessionize/app/src/main/java/co/touchlab/sessionize/NotificationPublisher.kt [hunk 1]
```diff
@@ -23,16 +23,18 @@ class NotificationPublisher : BroadcastReceiver() {
...
-            // notificationId is a unique int for each notification that you must define
-            this.notify(notificationId, notification)
-            Log.i(TAG,"Showing Local Notification $notificationId")
-        }
-        if(notificationId == notificationReminderId) {
-            NotificationsModel.recreateReminderNotifications()
-        }
-        if(notificationId == notificationFeedbackId){
-            NotificationsModel.recreateFeedbackNotifications()
+        notification?.let {
```
Comment (samhill303) [inline sessionize/app/src/main/java/co/touchlab/sessionize/NotificationPublisher.kt]: How could notification be null? don't you only send these intents with notifications?

Comment (KevinSchildhorn) [reply sessionize/app/src/main/java/co/touchlab/sessionize/NotificationPublisher.kt]: I'm not sure, we are making pending intents without notifications when we cancel, but that's the only time. Either the notification or the intent is null but they shouldnt be able to be null.

Comment (KevinSchildhorn) [reply sessionize/app/src/main/java/co/touchlab/sessionize/NotificationPublisher.kt]: The crash says attempting to read extras from null, which makes me think intent, but the crash is on line 28 which is the notify line.

File: sessionize/app/src/main/java/co/touchlab/sessionize/event/EventFragment.kt [hunk 2]
```diff
@@ -104,26 +104,28 @@ class EventFragment : Fragment() {
     }
 
     private fun updateContent(event: SessionInfo, formattedRoomTime:String) {
-        val adapter = EventDetailAdapter(activity!!)
+        if(::recycler.isInitialized) {
```
Comment (samhill303) [inline sessionize/app/src/main/java/co/touchlab/sessionize/event/EventFragment.kt]: Does this imply this is getting called before or after the view is initialized? Are we not shutting down the viewmodel properly?

**Pull request merged**